alembic.ini

```diff
@@ -0,0 +1,59 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = alembic
+
+# template used to generate migration files
```

Comment:
It's boiler plate... I'm gonna leave it in for now since I haven't personalized any of it.